### DISCUSSION TITLE
看完大佬更新之后的我。。。

### DISCUSSION THREAD

**Discussion by blibilijojo** [Richasy/Bili.Uwp#1112] [展示]
云之幻大佬，你是我的“神”！！！
![image](https://user-images.githubusercontent.com/102420734/164910691-60433013-f79d-4805-94cd-633690db41ce.png)

**Comment by blibilijojo** [comment]
bili漫游，弹幕引擎更新，文字动态，我已经high到不行了
![image](https://user-images.githubusercontent.com/102420734/164910914-0a23c12d-a3be-4365-bb26-1cdbdbee5140.png)

**Reply by Richasy** [reply]
那明天你就可以实际体验了🤣
